modules/serverless-create-channel-odc.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-create-channel-odc_{context}"]
+= Creating a channel using the Developer perspective
+
+You can create a channel with the cluster default configuration by using the {product-title} web console.
+
+.Prerequisites
+To create channels using the *Developer* perspective ensure that:
+
+* The {ServerlessOperatorName}, Knative Serving and Knative Eventing are installed on your {product-title} cluster.
+* You have logged in to the web console.
+* You are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+.Procedure
+
+. In the *Developer* perspective, navigate to *+Add* -> *Channel*.
```

Comment:
Just wanted to double-check whether the `+` is necessary in front of `Add` :slightly_smiling_face:

Reply:
I'd say yes because this is the literal name of the option in the UI?

Reply:
Yep, makes sense. I saw it several times later in the docs and figured that was the case.